app/src/main/java/fr/free/nrw/commons/feedback/FeedbackContentCreator.java

```diff
@@ -28,6 +30,10 @@ public FeedbackContentCreator(Context context, Feedback feedback) {
     public void init() {
         // Localization is not needed here, because this ends up on a page where developers read the feedback, so English is the most convenient.
 
+        LocalDateTime currentDateTime = LocalDateTime.now();
```

Comment:
**Call requires API level 26 (current min is 21): `java.time.LocalDateTime#now`**
**Call requires API level 26 (current min is 21): `java.time.format.DateTimeFormatter#ofPattern`**
**Call requires API level 26 (current min is 21): `java.time.LocalDateTime#format`**

This will throw error on API 26. You will need to use [desugaring](https://developer.android.com/studio/write/java8-support?authuser=2#library-desugaring) inorder to get this support towards to API 21. 

Ref : https://stackoverflow.com/questions/56695997/how-to-fix-call-requires-api-level-26-current-min-is-25-error-in-android

This will increase unnecessary refactoring this can be done with this 2 line of code.

```
        final SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.ENGLISH);
        String formattedDateTime = formatter.format(new Date());
```

Reply:
@Gupta-Suruchi Would you mind trying that? :-)